docs/HowSwiftImportsCAPIs.md

```diff
@@ -929,30 +922,14 @@ enum CardinalDirection {
 ```swift
 // C header imported in Swift.
 
-struct HomeworkExcuse : Equatable, RawRepresentable {
-  init(_ rawValue: UInt32)
-  init(rawValue: UInt32)
-  var rawValue: UInt32
-  typealias RawValue = UInt32
-}
-var EatenByPet: HomeworkExcuse { get }
-var ForgotAtHome: HomeworkExcuse { get }
-var ThoughtItWasDueNextWeek: HomeworkExcuse { get }
-
-enum InputDevice : UInt32 {
-  init?(rawValue: UInt32)
-  var rawValue: UInt32 { get }
-  typealias RawValue = UInt32
+enum InputDevice: CUnsignedInt {
```

Comment:
I would prefer to include all conformances to show the complete API that is available.